internal/balancergroup/balancergroup.go

```diff
@@ -278,6 +275,9 @@ func (bg *BalancerGroup) Start() {
 	}
 
 	for _, config := range bg.idToBalancerConfig {
+		if config.balancer == nil {
```

Comment:
Do this check in `startBalancer`?

Otherwise, will `Add()` panic due to nil?

Reply:
Yeah, I'll move it, but I don't think Add() will panic in this case.